src/apps/assets/sass/main.scss

```diff
@@ -119,10 +123,9 @@ $run-menu-height: 65px;
 
 .juttle-source {
     background-color: transparent;
-    color: inherit;
-    padding: 10px;
-    border-bottom: 1px solid $gray-dark;
-    @include flex-shrink(0);
+    border-style: solid;
+    border-width: 1px;
+    border-color: $gray-light;
```

Comment:
to be consistent with the other inserts, just do a `border-bottom: 1px solid $gray-dark`